qa/pull-tester/rpc-tests.py

```diff
@@ -37,7 +38,7 @@
     # terminal via ANSI escape sequences:
     BOLD = ('\033[0m', '\033[1m')
 
-RPC_TESTS_DIR = BUILDDIR + '/qa/rpc-tests/'
+RPC_TESTS_DIR = SRCDIR + '/qa/rpc-tests/'
```

Comment:
Wasn't it always (and still is) the case that the cache ends up in the current directory when the tests are launched?